examples/docker_aws_lambda_example/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM fedora:latest
+
+# Install pre-reqs
+RUN dnf install wget -y
+RUN dnf install gcc -y
+
+# Install librdkafka
+RUN rpm --import https://packages.confluent.io/rpm/5.4/archive.key
+RUN dnf install librdkafka-devel -y
+
+# Install Go v1.14
+RUN wget https://dl.google.com/go/go1.14.linux-amd64.tar.gz && tar -xvf go1.14.linux-amd64.tar.gz && rm go1.14.linux-amd64.tar.gz
```

Comment:
missing -z to tar (-xvzf) since it is a gzipped archive, or perhaps it understands that automatically?